src/lava/magma/runtime/_c_message_infrastructure/csrc/channel/dds/dds_channel.cc

```diff
@@ -18,8 +35,10 @@ DDSChannel::DDSChannel(const std::string &topic_name,
                                   dds_transfer_type,
                                   dds_backend,
                                   size);
-  send_port_ = std::make_shared<DDSSendPort>(dds_);
-  recv_port_ = std::make_shared<DDSRecvPort>(dds_);
+  send_port_ = std::make_shared<DDSSendPort>(
+            topic_name+"_src_" + std::to_string(std::rand()), 0, 0, dds_);
```

Comment:
This construct API is deprecated?
It doesn't make sense for set zero to `size` and `nbytes`.

Reply:
I modify the construct API of DDSSendPort and DDSRecvPort, because the old API didn't initialize the name, size, nbytes of AbstractSendPort. Lava cannot recognize the port name  if no name. for size and nbytes, we don't need them in dds ,but AbstractSend/RecvPort declares the variable. I think it shuold be initialized

Reply:
If old API is deprecated, pls remove it.

Reply:
The old constructor of DDSSend/RecvPort have be removed, and the old constructor DDSchannel is still retained here. In order to allow users to have two choices: they can define topic name according to themselves, or provide src_/dst_name like other channels.